ext/RMagick/rmimage.c

```diff
@@ -7419,10 +8100,17 @@ Image_import_pixels(int argc, VALUE *argv, VALUE self)
 
     if (!okay)
     {
+#if defined(IMAGEMAGICK_7)
+        CHECK_EXCEPTION()
```

Comment:
In below, `rm_magick_error()` will raise an exception. So it needs to destroy `exception` to avoid memory leak.

Reply:
Good catch!